src/components/PrivacyPolicy/index.tsx

```diff
@@ -0,0 +1,173 @@
+import { Trans } from '@lingui/macro'
+import Card, { DarkGreyCard } from 'components/Card'
+import Row, { AutoRow, RowBetween } from 'components/Row'
+import { useEffect, useRef } from 'react'
+import { ArrowDown, Info, X } from 'react-feather'
+import ReactGA from 'react-ga'
+import styled from 'styled-components/macro'
+import { ExternalLink, TYPE } from 'theme'
+import { isMobile } from 'utils/userAgent'
+
+import { useModalOpen, useTogglePrivacyPolicy } from '../../state/application/hooks'
+import { ApplicationModal } from '../../state/application/reducer'
+import { AutoColumn } from '../Column'
+import Modal from '../Modal'
+
+const Wrapper = styled.div`
+  max-height: 70vh;
+  overflow: auto;
+  padding: 0 1rem;
+`
+
+const StyledExternalCard = styled(Card)`
+  background-color: ${({ theme }) => theme.primary5};
+  padding: 0.5rem;
+  width: 100%;
+
+  :hover,
+  :focus,
+  :active {
+    background-color: ${({ theme }) => theme.primary4};
+  }
+`
+
+const HoverText = styled.div`
+  text-decoration: none;
+  color: ${({ theme }) => theme.text1};
+  display: flex;
+  align-items: center;
+
+  :hover {
+    cursor: pointer;
+  }
+`
+
+const StyledLinkOut = styled(ArrowDown)`
+  transform: rotate(230deg);
+`
+
+const EXTERNAL_APIS = [
+  {
+    name: 'Auto Router',
+    description: <Trans>The app fetches the optimal trade route from a Uniswap Labs server.</Trans>,
+  },
+  {
+    name: 'Infura',
+    description: <Trans>The app fetches on-chain data and constructs contract calls with an Infura API.</Trans>,
+  },
+  {
+    name: 'TRM Labs',
+    description: (
+      <Trans>
+        The app securely collects your wallet address and shares it with TRM Labs Inc. for risk and compliance reasons.
+      </Trans>
```

Comment:
This might be important. Maybe we should do this too? we should try to be compliant

https://github.com/gnosis/cowswap/blob/bbc64f12bb1773bb50c6e343c768a20c91d54ce6/src/hooks/useMonitoringEventCallback.ts#L23


In any case. It's important to at least create an issue, and block in a MOD the leaking of this information.
We should check with legal if this is important to add or not

Reply:
Added mod file for now